.buildkite/build-test-omnibus.sh

```diff
@@ -167,10 +167,11 @@ then
     echo "      chef-foundation-version: $CHEF_FOUNDATION_VERSION"
     echo "      config: omnibus/omnibus.rb"
     echo "      install-dir: \"/opt/chef\""
-    if [ $build_key == "mac_os_x-11-x86_64" ]
-    then
-      echo "      remote-host: buildkite-omnibus-$platform"
-    fi
+    if [ "$build_key" == "mac_os_x-12-x86_64" ]; then
+        echo "      remote-host: buildkite-omnibus-$platform"
+    elif [ "$build_key" == "mac_os_x-12-arm64" ]; then
+           echo "      remote-host: buildkite-omnibus-$platform"
```

Comment:
Spacing looks odd on this, but also, it looks like this logic could be collapsed down into a regex-style matcher:

```sh
if [[ "$build_key" =~ "mac_os_x-12-" ]] ; then
  echo " remote-host: buildkite-omnibus-$platform
fi
```